### PR TITLE
Feat: Add interim timeline message

### PR DESCRIPTION
**Pull request by tsm1820**
# Notion Task

https://www.notion.so/Interim-Message-for-empty-timeline-f374a5bd7da64ac682b5f3d1cf9fc146

# Changes Made In This PR

Added interim timeline message

### CODE REVIEW

**Review by millasml — CHANGES_REQUESTED** [review]
looking at the notion, i think there is a bunch more work to do:
image

you have to make the fake timeline entry like in there. i suggest you add the `className` prop to `TimelineEntry`, and put it in line 79. After that, you can just use `TimelineEntry` to create that fake one and override the styles easily

**Comment by gracuamole** [comment]
hi sheehui! don't need to create the fake example timeline entry for this task, as it's in a separate task! sorry for the confusion

**Comment by millasml** [comment]
thanks for the clarification grace! if that's the case, can we centre it?

image

File: src/pages/app/timeline.module.css [hunk 1]
```diff
@@ -54,4 +54,9 @@
 .promptMsg {
   margin-bottom: 15px;
   font-size: 15px;
+  text-align: center;
+}
+
+h1 {
+  text-align: center;
 }
```
Comment (millasml) [inline src/pages/app/timeline.module.css:62]: you can't do this, as it affects the rest of the h1 items:

image

File: src/pages/app/timeline.tsx [hunk 2]
```diff
@@ -183,7 +183,13 @@ const TimelinePage = ({ user }: { user: User }) => {
             <Grid item xs={12} className={timelineContainer}>
               <div className={timelineBox} ref={timelineRef}>
                 {entries.length === 0 ? (
-                  'Get started with a journal entry, activity or by creating a resume!'
+                  <div>
```
Comment (millasml) [inline src/pages/app/timeline.tsx:186]: create a new css class called `emptyTimeline`, and use it in line 186 over here. Over there, you can center the text. revert the text centering you did for h1 and promptMsg

**Review by millasml — CHANGES_REQUESTED** [review]

**Comment by millasml** [comment]
done in https://github.com/CareerSocius/resumebuilder/pull/204

**Pull request closed without merging**